Sources/CorePayments/Networking/UpdateClientConfigAPI.swift

```diff
@@ -0,0 +1,68 @@
+import Foundation
+
+/// This class coordinates networking logic for communicating with the /graphql?UpdateClientConfig API.
+@_documentation(visibility: private)
+public class UpdateClientConfigAPI {
+
+    // MARK: - Private Properties
+
+    private let coreConfig: CoreConfig
+    private let networkingClient: NetworkingClient
+
+    // MARK: - Initializer
+
+    public init(coreConfig: CoreConfig) {
+        self.coreConfig = coreConfig
+        self.networkingClient = NetworkingClient(coreConfig: coreConfig)
+    }
+
+    /// Exposed for injecting MockNetworkingClient in tests
+    init(coreConfig: CoreConfig, networkingClient: NetworkingClient) {
+        self.coreConfig = coreConfig
+        self.networkingClient = networkingClient
+    }
+
+    // MARK: - Internal Methods
+
+    public func updateClientConfig(orderID: String, fundingSource: String) async throws -> ClientConfigResponse {
+
+        let queryString = """
+            mutation UpdateClientConfig(
+                $orderID: String!,
+                $fundingSource: ButtonFundingSourceType!,
+                $integrationArtifact: IntegrationArtifactType!,
+                $userExperienceFlow: UserExperienceFlowType!,
+                $productFlow: ProductFlowType!,
+                $channel: ProductChannel!
+            ) {
+                updateClientConfig(
+                    token: $orderID
+                    fundingSource: $fundingSource
+                    integrationArtifact: $integrationArtifact,
+                    userExperienceFlow: $userExperienceFlow,
+                    productFlow: $productFlow,
+                    channel: $channel
+                )
+            }
+        """
+
+        let variables = UpdateClientConfigVariables(
+            orderID: orderID,
+            fundingSource: fundingSource,
+            integrationArtifact: "MOBILE_SDK",
+            userExperienceFlow: "INCONTEXT",
+            productFlow: "HERMES",
```

Comment:
```suggestion
            productFlow: "MOBILE_SDK",
```
Should this be `MOBILE_SDK` as well? I had to make this change on Android.

Reply:
No, we talked to Risk and Compliance and they wanted HERMES which is the default